ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -68,14 +76,12 @@ export default {
       delaySeconds: 2.1,
       durationSeconds: 5,
       alertText: (data, _, output) => output.text(),
+      tts: {
+        en: 'Go North/South Sides',
```

Comment:
tts nit, but you should also use output strings here, so that the tts string can be edited in the config.  Sorry that tests are not smart enough to tell you this.